src/patterns/SaleHeaderBanner/SaleHeaderBanner.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import { Meta } from '@storybook/react';
 import SaleHeaderBanner, { SaleHeaderBannerProps } from './SaleHeaderBanner';
-import { AuctionStatus } from '../../types/commonTypes';
+import { LotAuctionStatus } from '../../types/commonTypes';
```

Comment:
It does not make sense to change the name of this enum. SaleHeaderBanner logic is dependent on the state of the auction, not the individual lot.